default_translations/de/messages.de.yml

```diff
@@ -53,6 +55,19 @@ form:
   create-custom-alias: Erstelle Alias-Adresse
   create-random-alias: Generiere zufällige Alias-Adresse
   delete-alias: Alias-Adresse löschen
+  generate-recovery-token: Neuen Wiederherstellungscode generieren
+  recovery-token: Wiederherstellungscode
+  recovery-start: Prozess starten
```

Comment:
I would only call it "Zurücksetzen/Reset". "Process start" sound very technical and even not what the user want.

Reply:
Agreed, fixed now.